libweasyl/oauth.py

```diff
@@ -2,8 +2,31 @@
 from oauthlib.oauth2 import RequestValidator, WebApplicationServer
 
 from libweasyl import cache
+from libweasyl import security
+from libweasyl import staff
 from libweasyl.models.api import OAuthBearerToken, OAuthConsumer
 
+SCOPES = [
```

Comment:
Why not make this is a dictionary of scope names to dictionaries (or named tuples)? It would make `get_allowed_scopes()` simpler.